src/services/api/mock.tsx

```diff
@@ -0,0 +1,167 @@
+import axios, { AxiosRequestConfig } from 'axios';
+import { TUser } from './user/models';
+import { TUserSavedFilter } from './savedFilter/models';
+import { ApiResponse } from '.';
+import { SETS_API_URL } from './savedSet';
+import { IUserSetOutput } from './savedSet/models';
+import { USER_API_URL } from './user';
+import { SAVED_FILTER_API_URL } from './savedFilter';
+import { FENCE_NAMES } from 'common/fenceTypes';
+import { IFenceAclsPayload, IFenceAuthPayload } from './fence/models';
+import { FENCE_API_URL } from './fence';
+
+const apiInstance = axios.create();
+
+const sendRequestMock = async <T,>(config: AxiosRequestConfig, mockResponsed: any) => {
```

Comment:
I think it is best to remove it from the codebase

Reply:
I will remove it when we will migrate the user api